### DISCUSSION TITLE
[Q&A] 使用第三方模型，为什么插件没有生效？

### DISCUSSION THREAD

**Discussion by edisonzf2020** [lobehub/lobehub#875] [General Question] [answered]
### 💻 系统环境 | Operating System

Windows

### 🌐 浏览器 | Browser

Edge

### 🐛 问题描述 | Bug Description

选择自定义模型，插件都没有工作。
比如模型选择自定义的Gemini-pro，输入：现在北京的天气如何。插件没有工作。
如果选择openai的模型，插件有工作。

**Comment by leeyiding** [comment, marked as answer]
插件使用的function calling是OpenAI模型的特性，不支持第三方模型

**Comment by arvinxx** [comment]
三方模型如果兼容function call的逻辑，那么也是可以使用的。

[a comment by an automated account is not shown]
